Modules/Core/Common/test/itkImportImageTest.cxx

```diff
@@ -26,7 +27,7 @@ int
 itkImportImageTest(int, char *[])
 {
   // Create a C-array to hold an image
-  auto * rawImage = new short[8 * 12];
+  auto * rawImage = new short[static_cast<unsigned long>(8 * 12)];
```

Comment:
I am seeing unsigned long being used in a few places, which should not be a universally correct type.